src/llmcompressor/pipelines/registry.py

```diff
@@ -18,6 +18,7 @@
 __all__ = ["CalibrationPipeline"]
 
 SEQUENTIAL_MODIFIERS = (AWQModifier, GPTQModifier, SparsityModifierBase)
+NEED_DATA = (SmoothQuantModifier, *SEQUENTIAL_MODIFIERS)
```

Comment:
What about `CALIBRATED_MODIFIERS` to make name consistent with `SEQUENTIAL_MODIFIERS`?
```suggestion
CALIBRATED_MODIFIERS = (SmoothQuantModifier, *SEQUENTIAL_MODIFIERS)
```